zipline/algorithm.py

```diff
@@ -2234,6 +2235,22 @@ def set_max_leverage(self, max_leverage):
         control = MaxLeverage(max_leverage)
         self.register_account_control(control)
 
+    @api_method
+    def set_min_leverage(self, min_leverage, deadline):
+        """
+        Set a limit on the minimum leverage of the algorithm.
+
+        Parameters
+        ----------
+        min_leverage : float
+            The minimum leverage for the algorithm. If not provided there will
+            be no minimum.
+        deadline : datetime
```

Comment:
This might be more nicely expressed as a timedelta from the start of the algorithm instead of a datetime.